proto/prysm/v1alpha1/validator.proto

```diff
@@ -292,11 +292,6 @@ service BeaconNodeValidator {
         };
     }
 
-    
-    rpc SubmitValidatorRegistration(SignedValidatorRegistrationV1) returns (google.protobuf.Empty) {
-        option deprecated = true; // Use SubmitValidatorRegistrations
-    }
-
     rpc SubmitValidatorRegistrations(SignedValidatorRegistrationsV1) returns (google.protobuf.Empty) {
         option (google.api.http) = {
             post: "/eth/v1alpha1/validator/registration"
```

Comment:
should we update this to `validator/register_validator` to better match the specs? https://ethereum.github.io/beacon-APIs/#/Validator/registerValidator

Reply:
Sure. Didn't want to do that in this PR. Feel free to take it!